crates/proof-of-sql/src/sql/parse/query_context.rs

```diff
@@ -196,18 +196,8 @@ impl QueryContext {
         Ok(&self.res_aliased_exprs)
     }
 
-    pub fn get_order_by_exprs(&self) -> ConversionResult<Vec<OrderBy>> {
-        // Order by must reference only aliases in the result schema
-        for by_expr in &self.order_by_exprs {
-            self.res_aliased_exprs
-                .iter()
-                .find(|col| col.alias == by_expr.expr)
-                .ok_or(ConversionError::InvalidOrderBy {
-                    alias: by_expr.expr.as_str().to_string(),
-                })?;
-        }
-
-        Ok(self.order_by_exprs.clone())
+    pub fn get_order_by_exprs(&self) -> Vec<(usize, bool)> {
+        self.order_by_exprs.clone()
     }
```

Comment:
NIT: This isn't really doing any processing anymore, I wonder if we can make it return a reference too?
```suggestion
    pub fn get_order_by_exprs(&self) -> &[(usize, bool)] {
        &self.order_by_exprs
    }
```